### ISSUE TITLE
Cheats not working?

### ISSUE THREAD

**Issue by NelsonRosenberg** [libretro/RetroArch#4050]
I've a fresh install of retroarch on windows, downloaded the cheat files and not one of them work with any core. I saw some people saying only Game Genia codes work and not Action Replay so I'm not sure why we only get Action Replay codes in the cheats folder now. Do we need to do anything for Action Replay codes to work?

There also appears to be differences between encrypted and decrypted game genie codes, with some decrypted not working, but I'm not sure about the difference.

Tested with Genesis Plux, Snes9x, Bsnes-mercury, Picodrive in various games.

Thank you

**Comment by lasers** [comment]
_Beware of the free complimentary cheat codes supplied with untested ── not guaranteed, cheating claims!_

_That being said, the cheat codes are scrapped from various sites and are provided free of charge. Again, all untested._

_There are no quality assurance mechanisms involved. You can cheat, but you do so at your own peril. Cheat safe, my friend._

```
2089 Action Replay
1960 GameShark
1565 Game Genie
 669 Game Buster
 485 Code Breakers
 316 Xploder
```
_└── libretro-database/cht_

If you don't see Game Genie anywhere, you can try removing the old `cheats` directory and download again through `Online Updater`--> `Update Cheats`.

Feel free to report broken and/or working cheats. For me, some cheats will crash the game... Some cheats does not do anything... and some cheats just works great.

**Comment by NelsonRosenberg** [comment]
Alright, got the Game Genie codes now, might have been a problem with the download before. Buy yes, some codes do not work, maybe it's a problem with the rom itself that is incompatible with the codes. I see that Alladin, for the Genesis, has only the World version, but no the USA version, so probably these won't work.

A nice website for codes is http://gamehacking.org/, don't know if they would allow retroarch to get their library.

Thanks anyway lasers.

**Comment by lasers** [comment]
The World represent `USA, Europe, Japan (Part of Asia)`. The original file was just `Genesis/Pro Action Replay/Aladdin.cht`.

If the cheat codes mentioned certain region, then I slapped that region on the code. If the cheat codes does not mention anything, then I used all regions... later compressed into major regions and/or World... to better represents the games. Consider World generic because it came out in all regions so the cheat codes could be for all regions, USA, Europe, Japan, or any other region... or even none! Just a terrible code! 

You're welcome to try that code on all regions and see if it works on that particular game... then anybody can submit in the new change... from `Aladdin (World).cht` down to Aladdin (Europe).cht`. Feel free to submit in new working codes.
